hbase-server/src/main/java/org/apache/hadoop/hbase/HBaseServerBase.java

```diff
@@ -227,8 +227,13 @@ protected final void initializeFileSystem() throws IOException {
     // init the filesystem
     this.dataFs = new HFileSystem(this.conf, useHBaseChecksum);
     this.dataRootDir = CommonFSUtils.getRootDir(this.conf);
+    boolean tableDescriptorParallelLoadEnable =
+      conf.getBoolean("hbase.tabledescriptor.parallel.load.enable", false);
+    int tableDescriptorParallelLoadThreads =
+      conf.getInt("hbase.tabledescriptor.parallel.load.threads", 10);
```

Comment:
Is hbase.tabledescriptor.parallel.load.enable needed? If hbase.tabledescriptor.parallel.load.threads is configured as > 0, it means hbase.tabledescriptor.parallel.load.enable  is true, otherwise, it is false.